test/test.js

```diff
@@ -37,12 +37,17 @@ function getTestOptions (settings) {
     return defaults({}, settings && settings.set, disabled);
 }
 
+function writeConfig (from, to) {
+    writeFile(`${to}/.git/config`, readFile(`${to}/.git/config`) + readFile(`${from}/gitconfig`));
```

Comment:
This doesn't seem to work. I thought about adding some env variables (`GIT_CONFIG_NOGLOBAL=1 GIT_CONFIG_NOSYSTEM=1`) to disable using the global git configs, but `cp-sugar` doesn't handle these.

`exec('HELLO=1337 echo $HELLO').then(x => console.log(x));`